hugo/content/contribute.md

```diff
@@ -3,83 +3,233 @@ title: Contribute
 toc: true
 ---
 
-Mumble is a great program, but that is no reason to stop polishing it. There are numerous areas where the development team needs help, and if you feel like contributing, then any of the below is for you.
+You can contribute to the Mumble Project in multiple ways:
+
+<!-- Table of Content -->
+
+- [Report Bugs and suggest Improvements/Features](#report-bugs-and-feature-requests)
+- [Help with Development](#development)
+    - [Coding](#coding)
+        - [Additional projects](#additional-projects)
+    - Non-Coding: 
+        - [Translation](#translation)
+        - [Testing](#testing)
+        - [GUI/Mumble Themes](#mumble-theme)
+- Improve the [Documentation](#documentation) and [Website](#website)
+- [Support other users & take part in our community](#community)
+- [Host a Mumble server](#hosting)
+- [Promote Mumble](#promotion)
 
-# Provide Community Support
+---
+
+# Report bugs and Feature Requests
+
+1. Take a look at existing [issues in our GitHub repo](https://github.com/mumble-voip/mumble/issues) to check whether your issue or request was already reported.
+   If that's the case, then vote for it with a thumps up or add useful information & ideas with a comment.
+   
+2. If there is no issue report yet, then create a [new one](https://github.com/mumble-voip/mumble/issues/new/choose).
+
+**Note:** For security issues, see [Report a vulnerability]({{< relref "security#reporting-a-vulnerability" >}}).
+
+
+# Development
+
+We are always looking for helping hands willing to contribute to Mumble.
+
+## Coding
+
+Mumble is mostly written in **C++** using the **Qt** library.
+Thus previous knowledge in C++ and Qt is definitely useful, but we also encourage interested people who want to learn it to give it a try. We're happy to help you out, if needed.
+
+### Getting Started
+
+The best way to get into Mumble development is to link up with us on IRC ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)), also available via Matrix (Join: `#freenode_#mumble-dev:matrix.org` or use [Direct Link via Element.io (Web App)](#freenode_#mumble-dev:matrix.org)).
```

Comment:
Does clicking the matrix link actually work? 'cause it seems like it would not :thinking:

Reply:
Somehow it's the wrong link...will fix.